lib/open-uri-cached-archive.rb

```diff
@@ -1,5 +1,43 @@
 require 'open-uri-cached-archive/version'
+require 'yaml'
+require 'stringio'
+require 'open-uri'
 
 class OpenUriCachedArchive
-  # Your code goes here...
+  class CachedResponse
```

Comment:
I think we should be a little stricter about [one class per file](https://github.com/bbatsov/ruby-style-guide#one-class-per-file).

(I can't find a Rubocop rule that would enforce that, which I'm assuming is just me being blind, as it would be odd for them not to have something that simple…)